kedro_kubeflow/kfpclient.py

```diff
@@ -94,15 +94,15 @@ def compile(
         )
         self.log.info("Generated pipeline definition was saved to %s" % output)
 
-    def get_full_pipeline_name(self, pipeline_name):
-        return f"[{self.project_name}] {pipeline_name}"
+    def get_full_pipeline_name(self, pipeline_name, env):
+        return f"[{self.project_name}] {pipeline_name} (env: {env})"
```

Comment:
Is there any limit to the pipeline name length? If so we should maybe trim it?

Reply:
Good point. Documentation doesn't say anything: https://www.kubeflow.org/docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/#/definitions/apiPipeline

But the API throws:
```
HTTP response body: {"error_message":"Invalid pipeline name.: Invalid input error: Pipeline name too long. Support maximum length of 100","error_details":"Invalid pipeline name.: Invalid input error: Pipeline name too long. Support maximum length of 100"}
```